Election/Election/main.m

```diff
@@ -201,10 +203,48 @@ - (BOOL)pollsOpen {
 
 @end
 
+// Voting Simulator Class
+@interface VotingSimulator : NSObject
+
+-(void) simulate;
+
+@end
+
+@implementation VotingSimulator
+
+-(void) simulate{
+    
+    Election* election = [[Election alloc]init];
+    Contender* c = [[Contender alloc]init];
+    Contender* c2 = [[Contender alloc]init];
+    Contender* c3 = [[Contender alloc]init];
+    
+    [c initWithName:@"Carlos"];
+    [c2 initWithName:@"Carl"];
+    [c3 initWithName:@"Steven"];
+    
```

Comment:
222 - 224 is reainitializing your 3 contenders. You should use this method on lines 218-220.